tmcprototype/CspSubarrayLeafNode/test/CspSubarrayLeafNode_test.py

```diff
@@ -118,25 +130,33 @@ def test_ConfigureScan_invalid_json(self, tango_context):
         time.sleep(1)
         assert CONST.ERR_INVALID_JSON_CONFIG_SCAN in tango_context.device.activityMessage
 
-    def test_EndScan(self, tango_context):
-        """Test for EndScan"""
-        # PROTECTED REGION ID(CspSubarrayLeafNode.test_EndScan) ENABLED START #
-        test_input1 = '{"csp":{"frequencyBand":"1","delayModelSubscriptionPoint":"",' \
-                     '"visDestinationAddressSubscriptionPoint":"","fsp":[{"fspID":"1","functionMode":' \
-                     '"CORR","frequencySliceID":1,"integrationTime":1400,"corrBandwidth":0,' \
-                     '"channelAveragingMap":[]},{"fspID":"2","functionMode":"CORR","frequencySliceID":1,' \
-                     '"integrationTime":1400,"corrBandwidth":0,"channelAveragingMap":[]}]}}'
-        tango_context.device.ConfigureScan(test_input1)
-        time.sleep(3)
-        test_input2 = ['{"scanDuration": 10.0}']
-        tango_context.device.StartScan(test_input2)
-        time.sleep(2)
-        res = tango_context.device.EndScan()
-        tango_context.device.status()
-        time.sleep(1)
-        assert CONST.STR_ENDSCAN_SUCCESS in tango_context.device.activityMessage \
-               and res is None
-        # PROTECTED REGION END #    //  CspSubarrayLeafNode.test_EndScan
+    # def test_EndScan(self, tango_context):
```

Comment:
Add TODO or FIXME block else, remove the commented code.

Reply:
Done.